setup.py

```diff
@@ -28,6 +28,7 @@
     "redis": ["redis"],
     "oslo_messaging": ["oslo_messaging[amqp1]"],
     "zmq": ["pyzmq"],
+    "grpc": ["grpcio<=1.26.0", "grpcio-tools<=1.26.0", "protobuf"],
```

Comment:
tools is needed to generate the files but it's not packaged in rdo and is not a runtime dep.  it's more of a dev dep